server/base/src/main/java/org/apache/accumulo/server/util/ManagerMetadataUtil.java

```diff
@@ -199,7 +199,7 @@ public static void replaceDatafiles(ServerContext context, KeyExtent extent,
     TabletMutator tablet = context.getAmple().mutateTablet(extent);
 
     datafilesToDelete.forEach(tablet::deleteFile);
-    scanFiles.forEach(tablet::putScan);
+    scanFiles.stream().forEach(tablet::putScan);
```

Comment:
Curios why `.stream()` was added here.

Reply:
I had to change this before to getTabletFile() before changing putScan() to accept a StoredTabletFile so this was left from that, I will fix it and merge.